### ISSUE TITLE
square roots look funny in printed workbook

### ISSUE THREAD

**Issue by retabak** [bootstrapworld/curriculum#1294]
... but fine in workbook PDFs!

Images are from the first item on translate-arithmetic-to-circles-and-code-challenge.adoc from Order of Operations.

![image](https://user-images.githubusercontent.com/87144723/224831500-73e5b2d9-cb64-42c2-8fed-c02bb2527a17.png)

![image](https://user-images.githubusercontent.com/87144723/224831516-48c991d7-7408-4e76-b40b-e7d67c0fcf9c.png)

**Comment by retabak** [comment]
Huh! I think that sounds like a good move.

**Comment by retabak** [comment]
Adding a note that we need to also check the letter h in code font, which appeared teeny tiny on the printed version that we reviewed.

**Comment by retabak** [comment]
Funny, I kinda thought the distance formula looked okay, without the bar extending to the right. I was more bothered by tops of numbers getting chopped off!

I tried the live demo thing and my h looked okay?
![image](https://user-images.githubusercontent.com/87144723/230971691-4fe03347-54d8-4da8-8264-014efcf3a9f8.png)

**Comment by schanzer** [comment]
@retabak have you looked at the distance formula? It's nonsense without the bar extending.

In your screnshoot, the `g` looks small to me.

**Comment by retabak** [comment]
Yeah, but h is the only weird letter on the workbook that I reviewed?!

![image](https://user-images.githubusercontent.com/87144723/230973526-22d0ca73-7bd7-4d3f-9dc9-7baebdc4b34e.png)


Also, isn't this an acceptable way to depict a radical symbol? Without a bar extending? If I google "radical symbol" this is what I get: √ 

![image](https://user-images.githubusercontent.com/87144723/230973685-61f92e33-3fcc-46da-8e30-5737c0338162.png)

**Comment by retabak** [comment]
(That said, it'd be cool if the bar extended! Probably better, even! But I think this way is.... fine.)

**Comment by schanzer** [comment]
This is now fixed in #1318. I'll leave this issue open until the PR is merged.
